dev-libs/libunibreak/libunibreak-5.1.ebuild

```diff
@@ -16,7 +16,7 @@ SRC_URI+="
 
 LICENSE="ZLIB"
 SLOT="0/5"
-KEYWORDS="amd64 ~arm ~arm64 ~ppc x86"
+KEYWORDS="amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv x86"
```

Comment:
You can't just add arches willy-nilly. If this is going to be required, then the proper way to do it is file a bug for keywordreq.

Reply:
I added the extra keywords as unibreak is a new hard dep of Krita.
It seemed to me at the time like there were no indications that it wouldn't build on ppc64 or riscv. However this is of course best to check, but it seemed kinda bad to snub and users of ppc64 or riscv of the new version of krita.

I can only test on amd64, I think it would be a bad move to remove currently supported arches from Krita.

Of course I will remove keywords if you feel that is the appropriate thing to do.